mteb/tasks/Retrieval/eng/ChatDoctorRetrieval.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class ChatDoctorRetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="ChatDoctorRetrieval",
+        description="A medical retrieval task based on ChatDoctor_HealthCareMagic dataset containing 112,000 real-world medical question-and-answer pairs. Each query is a medical question from patients (e.g., 'What are the symptoms of diabetes?'), and the corpus contains medical responses and healthcare information. The task is to retrieve the correct medical information that answers the patient's question. The dataset includes grammatical inconsistencies which help separate strong healthcare retrieval models from weak ones. Queries are patient medical questions while the corpus contains relevant medical responses, diagnoses, and treatment information from healthcare professionals.",
+        reference="https://huggingface.co/datasets/embedding-benchmark/ChatDoctor_HealthCareMagic",
+        dataset={
+            "path": "embedding-benchmark/ChatDoctor_HealthCareMagic",
+            "revision": "50c2986fedffa33b38afd5c1752026f8e9e5ed1d",
+        },
+        type="Retrieval",
+        category="s2p",
+        modalities=["text"],
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="ndcg_at_10",
+        date=("2023-01-01", "2023-12-31"),
+        domains=["Medical"],
+        task_subtypes=["Question answering"],
+        license="not specified",
+        annotations_creators="expert-annotated",
+        dialect=[],
+        sample_creation="found",
+        bibtex_citation=r"""
+@misc{chatdoctor_healthcaremagic,
```

Comment:
This is referencing itself. Where did you get the data from? Maybe there's an associated paper?

Reply:
It's still not a paper and a rather generic org paper.